_posts/en/newsletters/2024-08-09-newsletter.md

```diff
@@ -0,0 +1,476 @@
+  They then replace the replacement with a transaction unrelated to
+  either user's version of the transaction.  When an LN channel has
+  unresolved [HTLCs][topic htlc], a successful replacement cycle can
+  allow the counterparty to steal from the honest party.
+
+  Using LN's current [anchor outputs][topic anchor outputs] channel
+  type, only a counterparty can perform a replacement cycling attack.
+  However, Todd points out that, because P2A allows anyone to spend the
+  output, anyone can perform a replacement cycling attack against
+  transactions using it.  Still, only a counterparty can economically
+  benefit from the attack, so there's no direct incentive for third
+  parties to attack P2A outputs.  The attack can be free in the case
+  where the attacker is planning to broadcast their own transaction at a
+  feerate higher than the honest user's P2A spend and the attacker
+  successfully completes the replacement cycle without their
+  intermediate state getting confirmed by miners.  All existing deployed
+  LN mitigations against replacement cycling attacks (see [Newsletter
+  #274][news274 cycle mitigate]) will be equally effective at defeating
+  P2A replacement cycling.
+
+- **Proposed BIP for scriptless threshold signatures:** Sivaram
+  Dhakshinamoorthy [posted][dhakshinamoorthy frost] to the Bitcoin-Dev
+  mailing list to announce the availability of a [proposed BIP][frost
+  sign bip] for creating [scriptless threshold signatures][topic
+  threshold signature] for Bitcoin's implementation of [schnorr
+  signatures][topic schnorr signatures].  This allows a set of signers
+  that have already performed a setup procedure (e.g. using
+  [ChillDKG][news312 chilldkg]) to securely create
+  signatures that only require interaction from a dynamic subset of
+  those signers.  The signatures are indistinguishable onchain from
+  schnorr signatures created by single-sig users and scriptless
+  multisignature users, improving privacy and fungibility.
+
+- **Optimistic verification of zero-knowledge proofs using CAT, MATT, and Elftrace:**
+  Johan T. Halseth [posted][halseth zkelf] to Delving Bitcoin to
+  announce that his tool, [Elftrace][], now has the ability to verify
+  zero-knowledge (ZK) proofs.  For this to be useful onchain, both
+  [OP_CAT][topic op_cat] and the [MATT][topic acc] proposed soft forks
+  would need to be activated.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review
+Club][] meeting, highlighting some of the important questions and
+answers.  Click on a question below to see a summary of the answer from
+the meeting.*
+
+  [# Add PayToAnchor(P2A), OP_1 <0x4e73> as standard output script for
+  spending][review club 30352] is a PR by [instagibbs][gh instagibbs]
+  that introduces a new `TxoutType::ANCHOR` output script type. Anchor
+  outputs have a `OP_1 <0x4e73>` witness program (resulting in a
```

Comment:
Witness program refers to just the data push part (`0x4e73`):
```suggestion
  outputs have a `OP_1 <0x4e73>` output script (resulting in a
```

Reply:
Ah right, thanks!